trinity/constants.py

```diff
@@ -16,9 +16,16 @@
 from p2p.kademlia import Address, Node
 
 
+# application identifier
+APP_IDENTIFIER_ETH1 = ""  # An empty identifier ensures backwards compatibility of data directory
```

Comment:
What are the implications of not preserving this backwards compat?  Just that previous PID files won't be in the *right* location or more than that?

Assuming the former, I'd prefer we go ahead and make this `eth1` rather than accumulate debt in an alpha codebase.

Reply:
The `chain` directory would turn into `chain-eth1` and the logs directory would turn into `logs-eth1`. So, not a big deal. We can put that in the release notes and people can move files manually if they care. That's fine with me if you prefer that.

Reply:
> That's fine with me if you prefer that.

:+1: